zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -56,7 +52,7 @@ private[zhttp] final case class Handler[R](
           )
         catch {
           case throwable: Throwable =>
-            writeResponse(
+            serverResponseWriter.write(
```

Comment:
```suggestion
            resWriter.write(
```

This is more concise.